odxtools/table.py

```diff
@@ -83,6 +85,8 @@ def __init__(
         table_rows: List[TableRow],
         long_name: Optional[str] = None,
         key_dop_ref: Optional[str] = None,
+        description: Optional[str] = None,
```

Comment:
this and the next line are probably meant to be a constructor arguments?

Reply:
ops, this looks like a spurious issue within some patch that I seem to have clicked upon...